### ISSUE TITLE
Can we add reinder support?

### ISSUE THREAD

**Issue by dheerajbhaskar** [timekit-io/booking-js#54]
Can a reminder mail be sent or a webhook called for events?

**Comment by laander** [comment]
@dheerajbhaskar Yup, we're currently working on some more powerful features for the "booking" oriented part of Timekit, which also includes reminders that are triggered with webhooks or by email. It's currently in internal alpha and i anticipate a release by the end of January/beginning of February.

I'll ping you when we have more news on it!

**Comment by dheerajbhaskar** [comment]
Sure, please do let me know. Thanks for the update.

On Thu, Dec 24, 2015 at 2:22 AM Lasse Boisen Andersen <
notifications@github.com> wrote:

> @dheerajbhaskar <https://github.com/dheerajbhaskar> Yup, we're currently
> working on some more powerful features for the "booking" oriented part of
> Timekit, which also includes reminders that are triggered with webhooks or
> by email. It's currently in internal alpha and i anticipate a release by
> the end of January/beginning of February.
>
> I'll ping you when we have more news on it!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/timekit-io/booking-js/issues/54#issuecomment-166988145>
> .
>
-- 
--- Disclaimer --- The information in this mail is confidential and is
intended solely for addressee. Access to this mail by anyone else is
unauthorised. Copying or further distribution beyond the original recipient
may be unlawful. Any opinion expressed in this mail is that of sender and
does not necessarily reflect that of OSW Technologies Pvt Ltd.---
